app/services/optout/OptOutProposition.scala

```diff
@@ -47,6 +48,13 @@ case class OptOutProposition(previousTaxYear: PreviousOptOutTaxYear,
       case _ => None
     }
   }
+
+  def expectedItsaStatuesAfter(customerIntent: TaxYear): Seq[ITSAStatus] = {
```

Comment:
Looks like a typo here from my copy 'n' paste Statues -> Statuses. Would you mind fixing it up?

Reply:
should be good now 😄